kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/WifiAccessPointImpl.java

```diff
@@ -141,4 +142,82 @@ public String toString() {
         }
         return sb.toString();
     }
+    
+    @Override
+	public int hashCode() {
```

Comment:
Could you explain what you mean exactly?

I generated this with the Eclipse IDE, and as far as I understand the equal method needs to be explicitly overridden, otherwise the comparison of the contents is not done (only references are compared)...